dds/DCPS/transport/rtps_udp/RtpsUdpDataLink.h

```diff
@@ -113,6 +113,8 @@ class OpenDDS_Rtps_Udp_Export RtpsUdpDataLink : public DataLink, public virtual
   /// Given a 'local' id, return the set of address for all remote peers.
   AddrSet get_addresses(const RepoId& local) const;
 
+  const RepoIdSet* updateWriterSeqReaders(const RepoId& writer, const SequenceNumber& seq);
```

Comment:
I'd rename this to make it more clear what the method is supposed to be doing. Update how?